### ISSUE TITLE
Secure apiKey & clientId

### ISSUE THREAD

**Issue by AndreiSirbu** [christiansmith/ngGAPI#7]
Hello,

I'm just starting to develop in AngularJS and I have a question(sorry if it's idiotic):
Is there a way to prevent the end user to find the aapiKey & clientId values from the js files?

Thanks.

**Comment by jcwatson11** [comment]
No. Security comes from the fact that you have restricted the domains that can use your javascript files in the [Google Developer Console](https://console.developers.google.com). First, you set up your project, then you set up ways that your project can access google API's. When setting up access for a new Browser Application, you would set up your domain access in the REFERERS field.
